siriuspy/siriuspy/search/id_search.py

```diff
@@ -18,23 +22,77 @@ class IDSearch:
         'CATERETE': 'SI-07SP:ID-APU22',  # titular: VPU19
         'EMA':      'SI-08SB:ID-APU22',  # titular: IVU18
         'MANACA':   'SI-09SA:ID-APU22',  # titular: 2 x APU22
-        'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
+        # 'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
```

Comment:
why not remove this line?

Reply:
see comment below

Reply:
but in this case the EPU will not be used on the SABIA beamline. maybe we don't need to keep this line here.

Reply:
ok, we could remove it.

Reply:
done